docs/README.md

```diff
@@ -1,10 +1,13 @@
 # Introduction
-This documentation describes the Event Logging XML Schema herein referred to as the 'schema'. This is a guide for anybody interested in using the schema for recording auditable events and includes details of the structure, features and common usage.
+This documentation describes the Event Logging XML Schema herein referred to as the 'schema'. This is a guide for 
```

Comment:
You seem to have split this line.  Lines should only be split for new paras, else they won't flow properly on different screen widths.

Reply:
fixed hard breaks in #26